### PR TITLE
Add Chinese Language

### PR DESCRIPTION
**Pull request by 239573049**
Signed-off-by: token <239573049@qq.com>

Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

### CODE REVIEW

**Comment by 239573049** [comment]
Yes, I finally passed the DCO

**Comment by 239573049** [comment]
@duglin How about you have a look

**Comment by 239573049** [comment]
I have access to all that
image

**Comment by duglin** [comment]
if you rebase on 'main' then the checks should work now

**Comment by 239573049** [comment]
> ```
> git checkout main
> git pull
> git checkout {your-branch}
> git rebase -i main
> git push {your-branch} -f
> ```
> 
> You may want to rename your branch to something other than "main" first

I have been on the mian branch, do I need to switch branches?

File: README.md [hunk 1]
```diff
@@ -109,10 +112,6 @@ native ecosystem by making our systems interoperable with CloudEvents.
 If there is a security concern with one of the specifications in this
 repository please [open an issue](https://github.com/cloudevents/spec/issues).
 
-A security assessment was performed by
```
[the remark this answers is not in the document]
Comment (duglin) [reply README.md:112]: and does it appear in the chinese version?

**Comment by duglin** [comment]
No you don't need to rename it unless you run into a naming conflict.

@JieDing I think this might be ready for review

**Comment by JieDing** [comment]
> No you don't need to rename it unless you run into a naming conflict.
> 
> @JieDing I think this might be ready for review

Cool, I'll review it.

File: README.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# CloudEvents
```
Comment (JieDing) [inline README.zh-cn.md:1]: There has been a [Chinese README.md](https://github.com/cloudevents/spec/tree/main/languages/zh-CN) already.

File: languages/zh-CN/README.md [hunk 3]
```diff
@@ -1,133 +1,27 @@
-# CloudEvents
+# CloudEvents 社区
```
Comment (JieDing) [inline languages/zh-CN/README.md:1]: Didn't see where this translation coming from

File: languages/zh-CN/SDK-PR-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
+# 拉取请求指导
```
Comment (JieDing) [inline languages/zh-CN/SDK-PR-guidelines.md:1]: It seems this translation already exists(/docs/SDK-maintainer-guidelines.md). Why do we have a redundant copy here?

File: languages/zh-CN/SDK-maintainer-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+# SDK 维护者指南
```
Comment (JieDing) [inline languages/zh-CN/SDK-maintainer-guidelines.md:1]: Ditto

**Comment by 239573049** [comment]
Can I merge?

**Comment by JieDing** [comment]
> Can I merge?

I don't think so, plz improve the translation quality. Don't directly use machine translation.

**Pull request closed without merging**